integration_tests/src/test/resources/flank_ios.yml

```diff
@@ -1,3 +1,6 @@
 gcloud:
   test: ../test_runner/src/test/kotlin/ftl/fixtures/tmp/earlgrey_example.zip
   xctestrun-file: ../test_runner/src/test/kotlin/ftl/fixtures/tmp/EarlGreyExampleSwiftTests_iphoneos13.4-arm64e.xctestrun
+  device:
+    - model: iphone8
+      version: 13.6
```

Comment:
should we force to use this device?

Reply:
I think we should use the default device. Thanks! Changed